### PR TITLE
fix code block syntax

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
compare http://symfony.com/doc/3.4/service_container.html#multiple-service-definitions-using-the-same-namespace with http://pr-9255-gshotaa-6qmocelev2lwe.eu.platform.sh/service_container.html#multiple-service-definitions-using-the-same-namespace to spot the difference

File: service_container.rst [hunk 1]
```diff
@@ -975,19 +975,19 @@ for classes under the same namespace:
...
-                resource: '../../src/Domain/*/CommandHandler'
-                tags: [command_handler]
-
-            event_subscribers:
-                namespace: App\Domain\
-                resource: '../../src/Domain/*/EventSubscriber'
-                tags: [event_subscriber]
+    # app/config/services.yml
+    services:
+	command_handlers:
```
Comment (javiereguiluz) [inline service_container.rst]: The indentation of these lines should be 8 white spaces instead of 1, right?

Comment (xabbuh) [reply service_container.rst]: good catch, fixed

**Comment by javiereguiluz** [comment]
Thanks Christian.

**Pull request merged**